README.md

```diff
@@ -100,6 +100,7 @@ Any donation is greatly appreciated :heart:
 - Form
 - Checkbox
 - Input
+- NuberInput
```

Comment:
Order alphabetically

Reply:
it's order alphabetically but there's a sub category for all the form components